data/service/api/v1/v1.go

```diff
@@ -20,6 +20,8 @@ func Routes() []service.Route {
 		service.Put("/v1/data_sets/:dataSetId", DataSetsUpdate, api.RequireAuth),
 		service.Get("/v1/time", TimeGet),
 		service.Post("/v1/users/:userId/data_sets", UsersDataSetsCreate, api.RequireAuth),
+
+		service.Get("/v1/partners/:partner/sector", PartnersSector),
```

Comment:
Given that in the OIDC this is referred to `Sector Identifier URI`, should this be called `sector_identifier`?

Reply:
I could go either way. The way I read the docs was as the sector's identifier url, not the sector identifier's url. Let me know which way you'd like to go.

Reply:
I think you're right